examples/hello_world/module/rust/src/lib.rs

```diff
@@ -38,9 +51,18 @@ struct Node {
 const STORAGE_NAME: &[u8] = b"HelloWorld";
 const FIELD_NAME: &[u8] = b"last-greeting";
 
+#[cfg(test)]
+fn init_logging() {
+    oak_tests::init_logging();
+}
+#[cfg(not(test))]
+fn init_logging() {
+    oak_log::init_default();
```

Comment:
Perhaps it would read better with https://doc.rust-lang.org/std/macro.cfg.html.

Reply:
Ooh, I didn't know about that, neat.

However, it doesn't appear to quite work here: the `oak_tests` import ends up still being needed for a non-test build.